webapp/src/pages/Food.js

```diff
@@ -95,7 +104,7 @@ function Offering({ id, description, image, closesAt }) {
               className="border rounded"
             />
             <div>
-              <Card.Link as={RLink} href={`/food/${id}`} className="h6 mb-0">
+              <Card.Link as={RLink} onClick={handleLink} className="h6 mb-0">
```

Comment:
Every single link should have an href. Failure to do so is inaccessible and one reason why people hate SPAs.